From ba6dbb228d6a3c2a75063aa2bc2edc94da18bc43 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 23 Feb 2018 22:48:50 +0100
Subject: [PATCH 068/152] workspace: Minor clarification

Code and comment were based on the old get_input_rect() and get_outer_rect()
method names that were changed to the more appropriate get_buffer_rect() and
get_frame_rect() a long time ago.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/713
---
 js/ui/workspace.js | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/js/ui/workspace.js b/js/ui/workspace.js
index b38a9428e..a8f57fd17 100644
--- a/js/ui/workspace.js
+++ b/js/ui/workspace.js
@@ -57,17 +57,16 @@ class WindowCloneLayout extends Clutter.LayoutManager {
         // has an extra set of "padding" around it that we need to trim
         // down.
 
-        // The outer rect (from which we compute the bounding box)
-        // paradoxically is the smaller rectangle, containing the positions
-        // of the visible frame. The input rect contains everything,
-        // including the invisible border padding.
-        let inputRect = window.get_buffer_rect();
+        // The bounding box is based on the (visible) frame rect, while
+        // the buffer rect contains everything, including the invisible
+        // border padding.
+        let bufferRect = window.get_buffer_rect();
 
         let box = new Clutter.ActorBox();
 
-        box.set_origin(inputRect.x - this._boundingBox.x,
-                       inputRect.y - this._boundingBox.y);
-        box.set_size(inputRect.width, inputRect.height);
+        box.set_origin(bufferRect.x - this._boundingBox.x,
+                       bufferRect.y - this._boundingBox.y);
+        box.set_size(bufferRect.width, bufferRect.height);
 
         return box;
     }
-- 
2.23.0

